src/blockstorage/blockleveldb.h

```diff
@@ -123,6 +128,7 @@ class CBlockLevelDB : public CDatabaseAbstract
         pwrapperblock->Sync();
         pwrapperundo->Sync();
     }
+    bool EraseBlock(const std::string &key);
```

Comment:
add doc comments to all new apis

Reply:
these apis are documented in the abstract class where i defined the virtual functions